t/plugin/key-auth.t

```diff
@@ -300,7 +300,45 @@ passed
 
 
 
-=== TEST 11: valid consumer
+=== TEST 11: customize querystring
```

Comment:
Please don't add the test in the middle of the other test. And we need to verify it via `GET /hello?xxx` in its own version of "valid consumer"